collector/prom-native-cockroachdb/collector.yml

```diff
@@ -0,0 +1,32 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'roach1'
+          scrape_interval: 3s
+          metrics_path: '/_status/vars'
+          params:
+            format: ['prometheus']
+          scheme: 'http'
+          tls_config:
+            insecure_skip_verify: true
+          static_configs:
+            - targets: ['roach1:8080', 'roach2:8080', 'roach3:8080']
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp:
+    endpoint: ingest.staging.lightstep.com:443
+    headers:
+             "lightstep-access-token": "ezKD2Lo3+5xVULCEAySlqnY1rTBefXN1Dj2CdGLfaAicYA3Fngu52zCIih93+73chQ81I2ax/vpGFJ2a+2+Nh0gWPHOsu0ypKRDNvNuN"
```

Comment:
We need to remove this ASAP. Probably best path from here with all things considered...
1. delete this branch and PR
2. disable the token in the app
3. check your ServiceNow email and Teams in case you need to reply to any alerts from the entropy scanner
4. open a new PR without the sensitive data - the environment variable pattern should work alright for now
5. make sure that the new PR is opened on a branch you create within the Lightstep repo and not coming from your fork - otherwise we will never see the green check for the CI action on the PR